rename-index.md

```diff
@@ -20,6 +20,45 @@ The user must have the `CREATE` [privilege](privileges.html) on the table.
 
 | Parameter | Description |
 |-----------|-------------|
-|  |  |
+| `IF EXISTS` | Rename the column only if a column of `current_name` exists; if one does not exist, do not return an error. |
```

Comment:
I don't see the `current_name` parameter in the diagram. Should this be `index_name`?